pytest_motor/plugin.py

```diff
@@ -51,45 +52,36 @@ async def root_directory(pytestconfig: PytestConfig) -> Path:
 async def mongod_binary(root_directory: Path) -> Path:  # pylint: disable=redefined-outer-name
     """Return a path to a mongod binary."""
     destination: Path = root_directory.joinpath('.mongod')
-    mongod_binary_filename = destination.joinpath(
-        'mongodb-linux-x86_64-ubuntu1804-4.4.6/bin/mongod')
+    mongod_binary_relative_path = 'mongodb-linux-x86_64-ubuntu1804-4.4.6/bin/mongod'
+    mongod_binary_filename = destination.joinpath(mongod_binary_relative_path)
 
     if not mongod_binary_filename.exists():
         download_url = 'https://fastdl.mongodb.org/linux/mongodb-linux-x86_64-ubuntu1804-4.4.6.tgz'
-        (tar_filename, _) = urllib.request.urlretrieve(download_url)
-
-        with tarfile.open(tar_filename) as tar:
+        async with aiohttp.ClientSession() as session:
+            async with session.get(download_url) as resp:
+                with open('binaries.tar', 'wb') as binary_file:
+                    # Read by chunks to avoid big RAM consumption
```

Comment:
This is cool, I wouldn't have thought of doing that.

Reply:
Mainly, this is aiohttp snippet. Another approach is to use aiofiles library, but this will add even more unnecessary dependencies. I'm not 100% sure, but maybe the best way is to use tempfile lib.